platform/viewer/src/routes/Mode/Mode.tsx

```diff
@@ -38,20 +38,41 @@ async function defaultRouteInit({
 
   unsubscriptions.push(instanceAddedUnsubscribe);
 
-  const { unsubscribe: seriesAddedUnsubscribe } = DicomMetadataStore.subscribe(
-    DicomMetadataStore.EVENTS.SERIES_ADDED,
-    ({ StudyInstanceUID, madeInClient }) => {
-      const studyMetadata = DicomMetadataStore.getStudy(StudyInstanceUID);
-      if (!madeInClient) {
-        HangingProtocolService.run(studyMetadata);
-      }
+  const { unsubscribe: seriesAddedUnsubscribe } = DisplaySetService.subscribe(
+    DisplaySetService.EVENTS.DISPLAY_SETS_CHANGED,
+    displaySets => {
+      if (!displaySets || !displaySets.length) return;
+      // The assumption is that the display set at position 0 is the first
```

Comment:
Bring back the `madeInClient` stuff, since you don't want this subscription to run on SR export (client side generated DS)

Reply:
Actually one only wants the rendering to occur the first time things arrive, which was what was previously causing problems.    The problem with doing it multiple times is that it messes up the display if it runs a subsequent time.  There are quite a few issues with that, but I don't think I can solve all of them.  The made in client stuff only applies to the added event, so what I will do for now is not fire the changed event if it is made int he client, assuming that is what is desired.

Reply:
There is already a comment in the display set service indicating it isn't quite right - what should be done is to have an event that fires once the study metadata is loaded, and fires once, then have the added/removed/changed events, which are on:
1.  When a new display set is added
2. When a display set is removed
3. When a display set is changed, but NOT when it is added or removed
I've changed the display set to not call the changed if made in client is set, under the assumption that it already knows about that, but it seems like a bad idea.